src/lib/apis/objkt/queries.ts

```diff
@@ -0,0 +1,26 @@
+import { gql } from '@apollo/client';
+
+import { fromFa2TokenSlug } from 'lib/assets/utils';
+
+export const buildGetCollectiblesQuery = (slugs: string[]) => {
+  const items = slugs.map(slug => fromFa2TokenSlug(slug));
+
+  return gql`
+    query MyQuery {
```

Comment:
Please try using query variables like here:
https://www.apollographql.com/blog/apollo-client/how-to-filter-and-search-using-variables-in-apollo-client/#refetching-and-filtering